scripts/install-tree-sitter-lib

```diff
@@ -52,9 +52,18 @@ dir_name=$(dirname "$BASH_SOURCE")
 "$dir_name"/update-version-symlinks
 "$dir_name"/download-tree-sitter --lazy
 
+# For the Makefile of tree-sitter
```

Comment:
Please try to explain as much as possible about why we do this and how we set the values of CC, AR, and STRIP.

Reply:
Antonin has updated with a quite clear comment I think now. :)